presto-mongodb/src/main/java/com/facebook/presto/mongodb/WriteConcernType.java

```diff
@@ -22,7 +22,7 @@
     ACKNOWLEDGED(WriteConcern.ACKNOWLEDGED),
     FSYNC_SAFE(WriteConcern.FSYNC_SAFE),
     FSYNCED(WriteConcern.FSYNCED),
-    JOURNAL_SAFEY(WriteConcern.JOURNAL_SAFE),
+    JOURNAL_SAFETY(WriteConcern.JOURNAL_SAFE),
```

Comment:
API?

Reply:
I think this should be JOURNAL_SAFE to match the mongo terminology.  This also needs a release note since it's changing the configuration options for a user facing config.